src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.cs

```diff
@@ -0,0 +1,1197 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Net.Sockets
+{
+    // TODO
+    // - ReceiveMessageFrom
```

Comment:
Not sure what you mean--this is already Linux/Unix specific. Are you just referring to changing the file name?